pytest-selenium/pages/content.py

```diff
@@ -0,0 +1,45 @@
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as expected
+
+from pages.base import Page
+from regions.base import Region
+
+
+class Content(Page):
+    URL_TEMPLATE = "/books/{book_slug}/pages/{page_slug}"
+
+    _main_content_locator = (
+        By.CSS_SELECTOR,
+        '#main-content > div > div > div h1[data-type="document-title"]',
```

Comment:
What are some reasons for using very-specific selectors in the Selenium tests?

```suggestion
        'h1',
```

If the WCAG guidelines are followed (I think we do so far) there should _only_ be one `h1` in the whole document. And if we don't follow the guidelines then something should blow up anyway 😄